using_images/other_images/jenkins.adoc

```diff
@@ -52,18 +52,95 @@ You can find https://github.com/openshift/origin/tree/master/examples/image-stre
 
 === Initializing Jenkins
 
+There are two methods for managing authentication with the {product-title} Jenkins images when they are initially started.
+
+* Standard authentication provided by Jenkins
+
+* {product-title} OAuth authentication provided by the OpenShift Login plugin.
+
+==== Jenkins standard authentication
+
+Jenkins authentication will be used by default with the {product-title} Jenkins images if the image is run outside of {product-title}.
+
 The first time you start Jenkins, the configuration is created along with the administrator user and password.
-The default login is `*admin/password*`.
+The default user login is `*admin/password*`.
 The default password can be configured by setting the `*JENKINS_PASSWORD*` environment variable.
 
 The following command creates a new Jenkins xref:../../architecture/core_concepts/pods_and_services.adoc#pods[pod] with Jenkins running in a container:
 
+If you choose to run with standard Jenkins authentication while the {product-title} Jenkins image runs inside of {product-title}, and you instantiate Jenkins from the template
+provided by {product-title}, the password for `*admin*` will be a randomly generated string.  The password will be noted in the output from the template instantiation.
+Otherwise, you can retrieve it by looking at the environment variable values defined on the Jenkins deployment configuration in the web console or by running:
+
+        $ oc env dc/jenkins --list | grep JENKINS_PASSWORD
+
```

Comment:
OK, I've pushed the corresponding update.  I reordered the presentation of the auth methods, putting oauth first.  I also added `oc new-app` examples for both ensuring oauth is the login method and ensuring standard jenkins is the standard method.  I think it flows better - but let me know.